### ISSUE TITLE
Input other than enum?

### ISSUE THREAD

**Issue by eitsupi** [devcontainers/feature-starter#28]
I noticed that the test case set value that are not in the enum.

https://github.com/devcontainers/feature-starter/blob/ea254a2b7d67d8526a6b9c85cfb5d96abfa780b2/test/_global/scenarios.json#L5-L7

https://github.com/devcontainers/feature-starter/blob/ea254a2b7d67d8526a6b9c85cfb5d96abfa780b2/src/color/devcontainer-feature.json#L7-L13

This may work for now, but I don't think it is generally expected behavior to enter any other value for an option that has enum set.

**Comment by samruddhikhandale** [comment]
Good point. I think the CLI should fail if the user uses some other value for an enum variable 👍 

ℹ️ Feature makes use of enum incorrectly in some places -https://github.com/search?q=repo%3Adevcontainers%2Ffeatures%20enum&type=code

Will fix that.
